pylint/checkers/deprecated.py

```diff
@@ -24,41 +24,58 @@
     nodes.ClassDef,
 )
 
+DEPRECATED_MODULE_MESSAGE: dict[str, MessageDefinitionTuple] = {
```

Comment:
Perhaps we should make this class variables? So they don't need to be imported together with the mixin.